.github/codecov.yml

```
@@ -0,0 +1,23 @@
+coverage:
+  # Display coverage percentage with 1 decimal point (XX.X%).
+  precision: 1
+  round: down
+
+  # FIXME: https://github.com/Kong/kubernetes-ingress-controller/issues/1552
+  # Ensure that the Codecov check is green regardless of the coverage number.
+  # This is because our current coverage pipeline is unstable and is
+  # causing fluctuations in coverage numbers being reported across runs.
+  # Also, I don't 100% believe that this is an effective way of communicating
+  # a coverage problem on a PR.
```

I generally agree with your sentiment, but I think a slight change in words may be helpful:

```suggestion
  # Also multiple maintainers at the time agree that this is not a 100% effective way
  # to communicate coverage problem on a PR.
```

---

I was expressing my own thoughts (hence the wording + traceability by `git blame`). If you contribute to that sentiment let's update the wording further to make it traceable:

```suggestion
  # Also @mflendrich and @shaneutt agree that this is not a 100% effective way
  # to communicate coverage problem on a PR.
```